engine/src/main/resources/assets/ui/menu/joinGameScreen.ui

```diff
@@ -1,338 +1,374 @@
 {
-    "type" : "joinGameScreen",
```

Comment:
Back to four spaces, please! 😛 I'd also recommend setting your Eclipse/IDEA codestyle template to Terasology's (located at `config/eclipse/formatting.xml`).